src/client/apps/edit/components/admin/components/article/article_authors.tsx

```diff
@@ -61,32 +55,31 @@ export class ArticleAuthors extends Component {
     const name = article.author ? article.author.name : ""
 
     return (
-      <Row>
-        <Col xs={6}>
-          <div className="field-group">
-            <label>Primary Author</label>
-            <input
-              className="bordered-input"
-              defaultValue={name}
-              onChange={e => this.onChangeAuthor(e.target.value)}
-            />
-          </div>
-        </Col>
+      <Flex flexDirection={["column", "row"]}>
+        <Box width={["100%", "50%"]} pb={40} pr={[0, 20]}>
```

Comment:
```
<Box width={["100%", "50%"]} pb={4} pr={[0, 2]}>
```

Reply:
The way `styled-system` works is it will 
1. take the value passed into the prop and see if it corresponds to a key in the theme file (corresponding to prop type), and then return the value
1. if it cant find the key, it will fall back to pixels